pyproject.toml

```diff
@@ -3,7 +3,7 @@
 requires = [
     "setuptools",
     "wheel",
-    "Cython>=0.28.5",
+    "Cython>=0.28.5,<3.0",
```

Comment:
Needed this to stop Cython 3.0 from installing with `PIP_PRE=true`

`PIP_PRE=true` is required because scipy 3.10 is only avaliable as a dev release in https://pypi.anaconda.org/scipy-wheels-nightly/simple